### PR TITLE
Css two column layout

### PR DESCRIPTION
**Pull request by t-doan**
css-two-column-layout

### CODE REVIEW

File: css-two-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+                facility, we support offline deployments, sync to the cloud and
+                run on any device 24/7.
+              </p>
+            </div>
+          </div>
+        </div>
+        <!-- columun-fourth 3 -->
+        <div class="column-fourth">
+          <div class="column-half">
+            <img class="icon" src="./images/data.svg" />
```
Comment (shawnkost) [inline css-two-column-layout/index.html:80]: Needs an `alt` attribute

File: css-two-column-layout/index.html [hunk 2]
```diff
@@ -23,7 +23,7 @@ <h1>Scale, Speed, Agility</h1>
       <div class="row">
         <div class="column-fourth">
           <div class="column-half">
-            <img class="icon" src="./images/software.svg" />
+            <img class="icon" src="./images/software.svg" alt="software" />
```
Comment (t-doan) [inline css-two-column-layout/index.html:26]: added alt

File: css-two-column-layout/index.html [hunk 3]
```diff
@@ -59,7 +59,10 @@ <h3>practice medicine, not administration</h3>
         <!-- columun-fourth 2 -->
         <div class="column-fourth">
           <div class="column-half">
-            <img class="icon" src="./images/circumstance.svg" />
+            <img
+              class="icon"
+              src="./images/circumstance.svg"
+              alt="circumstance" />
```
Comment (t-doan) [inline css-two-column-layout/index.html:65]: added alt

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**